sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js

```diff
@@ -2,7 +2,8 @@
   Copyright (c) Microsoft Corporation. All rights reserved.
   Licensed under the MIT Licence.
 
-  **NOTE**: If you are using version 1.1.x or lower, then please use the link below:
+  **NOTE**: This sample uses the preview of the next version of the @azure/service-bus package.
+  For samples using the current stable version of the package, please use the link below:
```

Comment:
Can we retain the reference to version numbers here?

> **NOTE**: This sample uses the preview of the next version (v7) of the @azure/service-bus package.
  For samples using the current stable version (v1) of the package, please use the link below:

Reply:
Done.